lib/chutney/TorNet.py

```diff
@@ -863,6 +871,15 @@ def getBridge(self):
         except KeyError:
             return 0
 
+    def getEd25519Id(self):
+        """Return the value of ed25519 key"""
+        try:
+            self._setEd25519Id()
+            return self._env['ed25519_id']
+        except KeyError:
+            
```

Comment:
You don't need this empty line here.
It's harder to read the code with this gap.